src/jupyterlabcontroller/exceptions.py

```diff
@@ -581,20 +604,18 @@ def to_slack(self) -> SlackMessage:
             Slack message suitable for posting as an alert.
         """
         message = super().to_slack()
-        obj = f"{self.kind} {self.namespace}" if self.namespace else self.kind
-        message.blocks.append(SlackTextBlock(heading="Object", text=obj))
+        if self.url:
+            text = f"{self.method} {self.url}" if self.method else self.url
```

Comment:
You can save one mention of self.url here at the cost of another vertical line but not really worth it.